### PR TITLE
[Bug] [Zeta] imap-storage-file module e2e error

### LINKED ISSUE #5236: [Bug] [Zeta] imap-storage-file module e2e error
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.

### What happened

imap-storage-file module e2e error

### SeaTunnel Version

dev
https://pipelines.actions.githubusercontent.com/serviceHosts/5f1d2e93-1249-42e2-b16d-ed31c598c5d5/_apis/pipelines/1/runs/61768/signedlogcontent/80?urlExpires=2023-08-07T08%3A06%3A07.4388650Z&urlSigningMethod=HMACV1&urlSignature=sN%2FmU3jzGF0B9ZTamYyzoJwiQGtlREiVjySaq6VhR8w%3D

### SeaTunnel Config

```conf
-
```

### Running Command

```shell
-
```

### Error Exception

```log
-
```

### Zeta or Flink or Spark Version

dev

### Java or Scala Version

_No response_

### Screenshots

image

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by sunxiaojian**
## Purpose of this pull request

close #5236

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by sunxiaojian** [comment]
@hailin0 PTAL

File: seatunnel-engine/seatunnel-engine-storage/imap-storage-plugins/imap-storage-file/pom.xml [hunk 1]
```diff
@@ -72,12 +72,26 @@
             <artifactId>awaitility</artifactId>
         </dependency>
 
+        <!-- Imap storage dependency package  -->
```
Comment (hailin0) [inline seatunnel-engine/seatunnel-engine-storage/imap-storage-plugins/imap-storage-file/pom.xml]: remove?

Comment (sunxiaojian) [reply seatunnel-engine/seatunnel-engine-storage/imap-storage-plugins/imap-storage-file/pom.xml]: > remove?

done

**Review by hailin0 — APPROVED** [review]

**Review by liugddx — APPROVED** [review]
+1

**Pull request merged**